src/pages/Account/Tabs/ModulesTab/Contract.tsx

```diff
@@ -683,16 +688,18 @@ function ContractForm({
                 <TextField label="signer" disabled fullWidth />
               ))}
             {fnParams.map((param, i) => {
+              // TODO: Need a nice way to differentiate between option and empty string
```

Comment:
Talking with @BriungRi , we would probably want a dropdown between some and none